### PR TITLE
Fix GithubPackageResolver for private repositories

### LINKED ISSUE #38: Support for private Github-repos
Hey, there is currently no support for private Github-Repositorys.
It is possible to add this feature?

Thank you in advance.

### PR DESCRIPTION
**Pull request by derech1e**
The updater now works fine with private repos.
Fixes #38.

### CODE REVIEW

File: Onova/Internal/HttpClientEx.cs [hunk 1]
```diff
@@ -82,9 +82,11 @@ public static HttpClient GetSingleton()
         public static async Task GetStreamAndCopyToAsync(this HttpClient client, string requestUri, Stream destination,
             IProgress<double>? progress = null, CancellationToken cancellationToken = default)
         {
-            using var response = await client.GetAsync(requestUri, HttpCompletionOption.ResponseHeadersRead, cancellationToken);
-            response.EnsureSuccessStatusCode();
+            var request = new HttpRequestMessage(HttpMethod.Get, requestUri);
```
Comment (Tyrrrz) [inline Onova/Internal/HttpClientEx.cs:85]: `using var` instead of `var`

**Review by Tyrrrz — CHANGES_REQUESTED** [review]
Thanks. Tiny comment.

File: Onova/Internal/HttpClientEx.cs (hunk 1 above)
Comment (derech1e) [reply Onova/Internal/HttpClientEx.cs]: Okay, I've changed this.

**Comment by Tyrrrz** [comment]
Thanks!

**Pull request merged**